prototypes/Extension Point Prototype/.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
Can we pull that up into the global `.gitignore` to simply exclude all folders called `bin`?